firmware/lm32/processor.c

```diff
@@ -300,6 +301,10 @@ static const struct video_timing video_modes[PROCESSOR_MODE_COUNT] = {
 
 };
 
+int processor_h_active ;
+int processor_v_active ;
```

Comment:
I defined them here to use them globally. But later found similar definition in the header file. Hence deleting. Also since these variables are global, don't need to pass them through function.